app/js/components/mixins/ModalEffects.react.js

```diff
@@ -0,0 +1,46 @@
+import React from 'react'
+
+let ModalEffects = InnerComponent => class extends React.Component {
```

Comment:
@andela-iamadi like mixins. This should only be used as a last resort and when its going to be used by different components throught out the app. I get the concept. but then you should not need to do your own manual mixings if you are using es6 with react. part of what is leading to the proposal es7 decorator. `This should only be used if there is no other way to implement it`. That its in a tutorial doesn't mean its a good practice.